raptor_dbw_can/src/DbwNode.cpp

```diff
@@ -848,9 +851,46 @@ void DbwNode::recvCAN(const can_msgs::msg::Frame::SharedPtr msg) {
                 }
             } break;
         }
+
+        // generates TireTemp messages and assign them to TireTempReport.
+        tire_temp_report_msg.front_left = DbwNode::generateTireTemp(fl_tire_temperature);
```

Comment:
my thought was to generate these in the `DbwNode::timerTireCallback()` function, as we only need the values to be set right before we publish them, instead of here, every time we get any CAN message of any type.